models/deepseek_v4_pro/prefill_fwd.py

```
@@ -1558,24 +1687,99 @@ def main():
 
+    prompt_ids = None
+    if args.prompt_file is not None:
+        from pathlib import Path
+
+        from tokenizers import Tokenizer
+
+        if args.tokenizer is None:
+            raise SystemExit("--prompt-file requires --tokenizer")
+        tokenizer = Tokenizer.from_file(args.tokenizer)
+        prompt_ids = tokenizer.encode(Path(args.prompt_file).read_text()).ids
+        bos_id = tokenizer.token_to_id("<｜begin▁of▁sentence｜>")
```

Ruff (0.16.1)

[warning] String contains ambiguous `｜` (FULLWIDTH VERTICAL LINE). Did you mean `|` (VERTICAL LINE)?

(RUF001)